cmd/api-linter/integration_test.go

```diff
@@ -94,12 +100,14 @@ func TestRules_DisabledByFileComments(t *testing.T) {
 	`
 
 	for _, test := range testCases {
-		disableInFile := fmt.Sprintf("// (-- api-linter: %s=disabled --)\n", test.rule)
-		proto := disableInFile + "\n" + test.proto
-		result := runLinter(t, proto, config)
-		if strings.Contains(result, test.rule) {
-			t.Errorf("Rule %q should be disabled by file comments", test.rule)
-		}
+		t.Run(test.testName, func(t *testing.T) {
+			disableInFile := fmt.Sprintf("// (-- api-linter: %s=disabled --)", test.rule)
```

Comment:
Remove "(--" and "--)".

Reply:
The common use case will include these tokens.